src/class-parsely.php

```diff
@@ -640,6 +640,11 @@ private function are_credentials_managed(): bool {
 			return false;
 		}
 
+		if ( isset( $credentials['is_managed'] ) &&
+				is_bool( $credentials['is_managed'] ) ) {
```

Comment:
`is_managed` will always be boolean so we can remove this condition.

```
is_bool( $credentials['is_managed'] )
```

Reply:
This is deliberate. Can we be that certain that no user error or bug could occur that makes this a non-boolean? I generally prefer to be cautious, even if the values are platform-provided.

Reply:
It's hard coded to `true` but if you think keeping this is find then keep it.